parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -236,3 +236,6 @@ sysctl_reload() {
         fi
     done
 }
+version_lte() {
```

Comment:
should the func name be `version_gte` ?

Reply:
No, because it's a `<=` comparator. When we use it, it's:
`version_lte 1.16 ${KUBERNETES_VERSION}` 
which *should* read as "version 1.16 is less than or equal to the current Kubernetes version," but I'm open to changing it if that's confusing.

Reply:
Got it, my thoughts are that from a code maintenance standpoint we are really using this function for >= type semantics, and so using a <= func (swapping the vals) is arguably less expressive.

From a human expressivity standpoint we are saying "do this thing for all k8s versions starting from k8s version _v_".

Reply:
I hear that. I had a `version_gt` func working and this renaming + swap args was the simplest path to get the behavior I wanted. I can refactor this.